components/Section/Section.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from 'react'
+
+interface Props {
+  className?: string
+  p?: string
+  px?: string
+  py?: string
+  pt?: string
+  pb?: string
+  pl?: string
+  pr?: string
+  backgroundColor?: string
```

Comment:
Optimizing with background color tailwind should work. we should not use hex or rgb directly